plone/namedfile/scaling.py

```diff
@@ -341,6 +360,15 @@ def getImageSize(self, fieldname=None):
     def guarded_orig_image(self, fieldname):
         return guarded_getattr(self.context, fieldname, None)
 
+    def getRetinaScales(self):
+        try:
+            from Products.CMFPlone.utils import getRetinaScales
+        except ImportError:
```

Comment:
Trying the import makes it easier to support earlier Plone versions that do not have this functionality. At Zest Software we may want to backport some of this to Plone 4.

We slightly refactored this in a065e173ad8b725403e5ee3a31904ea47f614494 to try the import only once at the top of the `scaling.py` file.